### ISSUE TITLE
Last update introduces a bug on vuescroll-panel

### ISSUE THREAD

**Issue by doctorsirius** [YvesCoding/vuescroll#31]
On the last update, vuescroll-panel seems to not be set correctly, and the **margin-right: -17px**, used to hide the browser scollbar and the **overflow-y: scroll**, don't appear until you hover the panel, you can force the browser scrollbar to appear by refreshing the browser, be sure your mouse cursor is not hovering the panel.

That bug forced me to specify two those properties on my styles in order to fix it.

Chrome version: 61 (I'm using Electron)
OS: Windows

**Comment by wangyi7099** [comment]
What's is the version of your vuescroll?

**Comment by doctorsirius** [comment]
4.5.27

**Comment by wangyi7099** [comment]
I think this is not the bug of vuescroll, it's Vue's bug instead.  vuescroll shows/hides scrollbars by calculating the size of content and scroll panel in Vue's `mounted` hook. But in some cases or some browsers, Vuescroll cannot get **correct** size of content and scroll panel in the `mounted` hook of vuescroll instance.   See detail in [#7707](https://github.com/vuejs/vue/issues/7707). What the solution is  to update vuescroll's scrollbar again in some delay times. But it's a 'fast-solution', so I also have no ideas to do that...

**Comment by doctorsirius** [comment]
The thing is I've managed to fix it by setting **margin-right: -17px** and **overflow-y: scroll** on the **.vuescroll-panel** class. So you could test it a bit and introduce that properties without wait to hover. Did you reproduce the bug?

**Comment by wangyi7099** [comment]
1. Update to the latest version.
2. See the FAQ section in readme.
3. See my reply above.

**Comment by doctorsirius** [comment]
Ok, thanks.

**Comment by wangyi7099** [comment]
@doctorsirius 
Hi! Do you have the reproduce code for this bug?  If there is, I will study into it.

**Comment by wangyi7099** [comment]
Refactor some codes for this bug, can have a try again in next release.

**Comment by wangyi7099** [comment]
Try `4.5.32` @doctorsirius

**Comment by doctorsirius** [comment]
Hi! Sorry for the delay, I was busy, many thanks for the updates.

I will try to explain how I had to setup things to deal with some problems with your previous version of your component, sadly I'm not a developer so I can only explain you what I had to do.

- First, I had a problem with the horizontal scrolling appearing with no reason, that forced my to set **scrollOptions: { scrollpanel: { scrollingX: false, scrollingY: true } }** on the component I use the scroll. Now, with your last version I've removed it and haven't the horizontal scrolling issue.
- Second, with the previous update, the one you made on 28th (I think) and having the previous scrollOptions set in the same way, I had to add **margin-right: -17px !important** and o**verflow-y: scroll !important** to **.vuescroll-panel**, if not, I got the issue with the native Chrome scrollbar appearing on refresh, until hovering. Now, by removing the scrollOptions stuff there is no need to set .vuescroll-panel, now works.

So, if I use scrollOptions as I described I still get the issue, unless I set the .vuescroll-panel to the properties I told you. I understand that I'm telling the component to make vertical scroll appear, by setting scrollingY to true but that shouldn't be the problem, with that I just telling your component to force scroll to appear, but the component styled scroll, not the Chrome native one, and as I told you it appears just on refresh, the moment you hover the panel the native scrollbar dissappear.

Conclusion: maybe the problem comes when setting scrollOptions, maybe something is not initialized.

For me, now that I can remove those scrollOptions the problems are gone, so for me the issue is fixed, but it will still happen if someone set scrollOptions.

Hope that helps.
